packages/react-admin-core/src/Admin.js

```diff
@@ -0,0 +1,139 @@
+import React, { createElement } from 'react';
```

Comment:
git considers this file as a creation, while you mostly moved it. This will mess up the history for blames.

Reply:
Any solution ?